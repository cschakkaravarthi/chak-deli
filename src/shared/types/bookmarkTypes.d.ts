import Bookmark from '../models/Bookmark.model';
export declare const LOAD_BOOKMARKS = "LOAD_BOOKMARKS";
export declare const FETCH_BOOKMARKS = "FETCH_BOOKMARKS";
export declare const SET_BOOKMARK = "SET_BOOKMARK";
export declare const SAVE_BOOKMARK = "SAVE_BOOKMARK";
export declare const DELETE_BOOKMARK = "DELETE_BOOKMARK";
export declare const REMOVE_BOOKMARK = "REMOVE_BOOKMARK";
export interface LoadBookmarksAction {
    type: typeof LOAD_BOOKMARKS;
    bookmarks: Bookmark[];
}
export interface SetBookmarkAction {
    type: typeof SET_BOOKMARK;
    success: boolean;
}
export interface DeleteBookmarkAction {
    type: typeof DELETE_BOOKMARK;
    success: boolean;
}
export interface BookmarkState {
    bookmarks: Bookmark[];
    success: boolean;
}
//# sourceMappingURL=bookmarkTypes.d.ts.map