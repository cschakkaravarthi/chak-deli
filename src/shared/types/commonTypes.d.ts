export declare const TOGGLE_GLOBAL_TOAST = "TOGGLE_GLOBAL_TOAST";
export interface ToggleGlobalToastAction {
    type: typeof TOGGLE_GLOBAL_TOAST;
    displayToast: boolean;
    toastMessage: string;
    toastError: boolean;
}
export interface CommonState {
    displayToast: boolean;
    toastMessage: string;
    toastError: boolean;
}
//# sourceMappingURL=commonTypes.d.ts.map