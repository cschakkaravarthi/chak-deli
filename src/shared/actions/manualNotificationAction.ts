import { AnyAction } from 'redux';
import { ThunkDispatch } from 'redux-thunk';

import {
  LoadManualNotificationsAction,
  LOAD_MANUAL_NOTIFICATIONS,
  FETCH_MANUAL_NOTIFICATIONS,
  LoadManualNotificationsFilterAction,
  FETCH_MANUAL_NOTIFICATIONS_FILTER,
  SetFilterFacetAction,
  SET_MANUAL_NOTIFICATIONS_FILTER_FACET,
  LOAD_MANUAL_NOTIFICATIONS_FILTER,
  LOAD_MANUAL_NOTIFICATIONS_BELL,
  FETCH_MANUAL_NOTIFICATIONS_BELL,
  LoadManulNotificationsBellAction,
  SetIsNewOrArchived,
  SET_MANUAL_NOTIFICATIONS_IS_NEW_OR_ARCHIVED
} from '../types/manualNotificationTypes';

import ApiService from '../services/apiService';
import { ManualNotificationListModel, Categories, NotificationBell } from '../models/ManualNotification.model';
import { triggerToast } from './commonActions';
import { eventGA } from '../../utils/googleAnalytics';
import Facet from '../models/Facet.model';

export const loadManualNotifications = (
  notificationsList: ManualNotificationListModel
): LoadManualNotificationsAction => ({
  notificationsList,
  type: LOAD_MANUAL_NOTIFICATIONS
});

export const fetchManualNotifications = (newOrArchive: string, id?: string, type?: string) => {
  return async (dispatch: ThunkDispatch<{}, {}, AnyAction>): Promise<void> => {
    dispatch(() => ({ type: FETCH_MANUAL_NOTIFICATIONS }));
    return ApiService.getManualNotifications(notifications =>
      dispatch(loadManualNotifications(notifications)),
    newOrArchive,
    id || '', type || ''
    );
  };
};

export const loadManualNotificationsFilter = (
  filterList: Categories[]
): LoadManualNotificationsFilterAction => ({
  filterList,
  type: LOAD_MANUAL_NOTIFICATIONS_FILTER
});

export const fetchManualNotificationsFilter = () => {
  return async (dispatch: ThunkDispatch<{}, {}, AnyAction>): Promise<void> => {
    dispatch(() => ({ type: FETCH_MANUAL_NOTIFICATIONS_FILTER }));
    return ApiService.getManualNotificationsFilterList(filterLists =>
      dispatch(loadManualNotificationsFilter(filterLists))
    );
  };
};

export const setFilterFacet = (
  selectedFacet: Facet
): SetFilterFacetAction => ({
  selectedFacet,
  type: SET_MANUAL_NOTIFICATIONS_FILTER_FACET
});

export const loadNotificationsBell = (
  NotificationBell: NotificationBell
): LoadManulNotificationsBellAction => ({
  NotificationBell,
  type: LOAD_MANUAL_NOTIFICATIONS_BELL
});

export const setIsNewOrArchived = (
  isNewOrArchived: string
): SetIsNewOrArchived => ({
  isNewOrArchived,
  type: SET_MANUAL_NOTIFICATIONS_IS_NEW_OR_ARCHIVED
});

export const fetchNotificationsCount = () => {
  return async (dispatch: ThunkDispatch<{}, {}, AnyAction>): Promise<void> => {
    dispatch(() => ({ type: FETCH_MANUAL_NOTIFICATIONS_BELL }));
    return ApiService.getNotificationsBell(notifications =>
      dispatch(loadNotificationsBell(notifications))
    );
  };
};

export const setNotificationsArchive = (drupal_id: number | undefined) => {
  return (dispatch: ThunkDispatch<{}, {}, AnyAction>): void => {
    ApiService.setNotificationsArchive(
      () => {
        eventGA({
          category: 'Notification',
          action: 'Archived successfully'
        });
        dispatch(fetchNotificationsCount());
        dispatch(fetchManualNotifications('new'));
        dispatch(triggerToast('Notification archived successfully.'));
      },
      () => {
        dispatch(triggerToast('Error archiving notification.', true));
      },
      drupal_id
    );
  };
};

export const setNotificationsApprove = (sysId: string) => {
  return (dispatch: ThunkDispatch<{}, {}, AnyAction>): void => {
    ApiService.setNotificationsApprove(
      (state: string) => {
        if (state === 'approved') {
          dispatch(fetchNotificationsCount());
          dispatch(fetchManualNotifications('new'));
          dispatch(triggerToast('Request Approved successfully.'));
        } else {
          dispatch(triggerToast('Sorry, there was an error approving this request. We are looking into it.', true));
        }
      },
      () => {
        dispatch(triggerToast('Sorry, there was an error approving this request. We are looking into it.', true));
      },
      sysId
    );
  };
};

export const setNotificationsReject = (sysId: string, message: string) => {
  return (dispatch: ThunkDispatch<{}, {}, AnyAction>): void => {
    ApiService.setNotificationsReject(
      (state: string) => {
        if (state === 'rejected') {
          dispatch(fetchNotificationsCount());
          dispatch(fetchManualNotifications('new'));
          dispatch(triggerToast('Request Rejected successfully.'));
        } else {
          dispatch(triggerToast('Sorry, there was an error rejecting this request. We are looking into it.', true));
        }
      },
      () => {
        dispatch(triggerToast('Sorry, there was an error rejecting this request. We are looking into it.', true));
      },
      sysId,
      message
    );
  };
};
