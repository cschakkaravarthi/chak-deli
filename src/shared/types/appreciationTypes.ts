import { AppreciationCardNotificationModel as AlertModel } from '../models/Notification.model';
import { AppreciationCardTemplate, AppreciationCardsListGrouped } from '../models/AppreciationCards.model';

export const FETCH_APPRECIATION_DATA = 'FETCH_APPRECIATION_DATA';
export const LOAD_APPRECIATION_ALERT = 'LOAD_APPRECIATION_ALERT';
export const LOAD_APPRECIATION_ALERT_ERROR = 'LOAD_APPRECIATION_ALERT_ERROR';
export const CLEAR_APPRECIATION_REDUCER = 'CLEAR_APPRECIATION_REDUCER';
export const LOAD_APPRECIATION_MANAGER_ALERT = 'LOAD_APPRECIATION_MANAGER_ALERT';
export const SET_NEW_CARD_DETAIL = 'SET_NEW_CARD_DETAIL';
export const FETCH_CARD_TEMPLATES = 'FETCH_CARD_TEMPLATES';
export const CLEAR_NEW_CARD_FORM = 'CLEAR_NEW_CARD_FORM';
export const TOGGLE_SUCCESS_SENT_ALERT = 'TOGGLE_SUCCESS_SENT_ALERT';
export const GET_APPRECIATION_CARDS_LIST_GROUPED = 'GET_APPRECIATION_CARDS_LIST_GROUPED';
export const VIEW_APPRECIATION_CARD = 'VIEW_APPRECIATION_CARD';
export const APPRECIATION_CARD_ID = 'APPRECIATION_CARD_ID';
export const ARCHIVE_ALERT_STATE = 'ARCHIVE_ALERT_STATE';
export const LOAD_CARDS_IN_GROUP = 'LOAD_CARDS_IN_GROUP';

export interface AppreciationState {
  cardId: string;
  cardArchiveSuccess: boolean;
  isFetching: boolean;
  appreciationCardAlert?: AlertModel[];
  appreciationCardAlertError?: boolean;
  appreciationManagerCardAlert?: AlertModel[];
  newAppreciationCardForm: NewAppreciationCardForm;
  cardTemplates: AppreciationCardTemplate[];
  showSentSucessAlert: boolean;
  appreciationCardsListGrouped?: AppreciationCardsListGrouped;
  cardToView?: AlertModel;
  displayCardView: boolean;
}

export interface NewAppreciationCardForm {
  senderName?: string;
  templateId?: string;
  cardMessage?: string;
  recipientEmail?: string;
  recipientLastName?: string;
  recipientFirstName?: string;
  recipientSupervisorEmail?: string;
  managerName?: string;
}

export interface ArchiveAlertState {
  cardArchiveSuccess: boolean;
  type: typeof ARCHIVE_ALERT_STATE;
}

export interface SetArchiveAlert {
  cardId: string;
  type: typeof APPRECIATION_CARD_ID;
}

export interface FetchRequest {
  isFetching: boolean;
  type: typeof FETCH_APPRECIATION_DATA;
}

export interface LoadAppreciationAlertAction {
  isFetching: boolean;
  type: typeof LOAD_APPRECIATION_ALERT;
  appreciationCardAlert: AlertModel[];
}

export interface LoadAppreciationManagerAlertAction {
  isFetching: boolean;
  type: typeof LOAD_APPRECIATION_MANAGER_ALERT;
  appreciationManagerCardAlert: AlertModel[];
}

export interface AppreciationCardAlertError {
  isFetching: boolean;
  type: typeof LOAD_APPRECIATION_ALERT_ERROR;
  appreciationCardAlertError?: boolean;
}

export interface ClearAprreciationAction {
  type: typeof CLEAR_APPRECIATION_REDUCER;
}

export interface SetNewCardDetailAction {
  type: typeof SET_NEW_CARD_DETAIL;
  payload: NewAppreciationCardForm;
}

export interface SetDetailGroupAction {
  type: typeof SET_NEW_CARD_DETAIL;
  payload: NewAppreciationCardForm;
}

export interface FetchAppreciationCardTemplatesAction {
  type: typeof FETCH_CARD_TEMPLATES;
  cardTemplates: AppreciationCardTemplate[];
}

export interface ClearNewCardFormAction {
  type: typeof CLEAR_NEW_CARD_FORM;
}

export interface ToggleSentSuccessAlert {
  type: typeof TOGGLE_SUCCESS_SENT_ALERT;
  showSentSucessAlert: boolean;
}

export interface GetAppreciationCardsListGroupedAction {
  type: typeof GET_APPRECIATION_CARDS_LIST_GROUPED;
  appreciationCardsListGrouped: AppreciationCardsListGrouped;
}

export interface ViewAppreciationCardAction {
  type: typeof VIEW_APPRECIATION_CARD;
  displayCardView: boolean;
  cardToView?: AlertModel;
}

export interface AddAppreciationCardsToGroupAction {
  type: typeof LOAD_CARDS_IN_GROUP;
  appreciationCards: AlertModel[];
  cardsType: string;
}
