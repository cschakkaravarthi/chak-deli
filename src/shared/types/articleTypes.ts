import Article from '../models/Article.model';
import Facet, { FacetProperty } from '../models/Facet.model';
import { VariousContentModel } from './contentTypes';

export const LOAD_ARTICLES = 'LOAD_ARTICLES';
export const LOAD_ARTICLE_BY_ID = 'LOAD_ARTICLE_BY_ID';
export const FETCH_MORE_ARTICLES = 'LOAD_MORE_ARTICLES';
export const SET_ARTICLE_FILTER_FACET = 'SET_ARTICLE_FILTER_FACET';
export const CLEAR_ARTICLES_LIST = 'CLEAR_ARTICLES_LIST';
export const LIKE_ARTICLE_TOGGLE = 'LIKE_ARTICLE_TOGGLE';

export interface ArticleState {
  article?: Article;
  isLastPage: boolean;
  articleList: Article[];
  facets: Facet;
  selectedFacet: string;
  pageCount: number;
}

export interface LoadArticlesAction {
  type: typeof LOAD_ARTICLES;
  articleList: Article[];
}

export interface LoadArticleByIdAction {
  type: typeof LOAD_ARTICLE_BY_ID;
  article: VariousContentModel;
}
export interface LoadMoreArticlesAction {
  type: typeof FETCH_MORE_ARTICLES;
  articleList?: Article[];
  facets?: Facet;
}
export interface SetFilterFacetAction {
  type: typeof SET_ARTICLE_FILTER_FACET;
  selectedFacet: FacetProperty;
}

export interface ClearArticlesListAction {
  type: typeof CLEAR_ARTICLES_LIST;
}

export interface LikeArticleToggleAction {
  type: typeof LIKE_ARTICLE_TOGGLE;
  articleId: number;
  likedByUser: boolean;
}
