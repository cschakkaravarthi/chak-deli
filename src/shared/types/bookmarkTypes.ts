import Bookmark from '../models/Bookmark.model';

export const LOAD_BOOKMARKS = 'LOAD_BOOKMARKS';
export const FETCH_BOOKMARKS = 'FETCH_BOOKMARKS';
export const SET_BOOKMARK = 'SET_BOOKMARK';
export const SAVE_BOOKMARK = 'SAVE_BOOKMARK';
export const DELETE_BOOKMARK = 'DELETE_BOOKMARK';
export const REMOVE_BOOKMARK = 'REMOVE_BOOKMARK';

export interface LoadBookmarksAction {
  type: typeof LOAD_BOOKMARKS;
  bookmarks: Bookmark[];
}

export interface SetBookmarkAction {
  type: typeof SET_BOOKMARK;
  success: boolean;
}

export interface DeleteBookmarkAction {
  type: typeof DELETE_BOOKMARK;
  success: boolean;
}

export interface BookmarkState {
  bookmarks: Bookmark[];
  success: boolean;
}
