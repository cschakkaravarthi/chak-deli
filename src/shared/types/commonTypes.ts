import RejectModal from './../models/RejectModal';

export const CURRENT_USER_NAME = 'CURRENT_USER_NAME';
export const TOGGLE_GLOBAL_TOAST = 'TOGGLE_GLOBAL_TOAST';
export const SET_REJECT_MODAL_TEXTAREA = 'SET_REJECT_MODAL_TEXTAREA';
export const SET_REJECT_MODAL_ID_ISSHOW = 'SET_REJECT_MODAL_ID_ISSHOW';

export interface ToggleGlobalToastAction {
  type: typeof TOGGLE_GLOBAL_TOAST;
  displayToast: boolean;
  toastMessage: string;
  toastError: boolean;
}

export type CurrentUser = {
  family_name: string;
  given_name: string;
};

export interface SetCurrentUsername {
  type: typeof CURRENT_USER_NAME;
  currentUsername: CurrentUser;
}

export interface CommonState {
  currentUsername: CurrentUser;
  displayToast: boolean;
  toastMessage: string;
  toastError: boolean;
  idAndIsShow: RejectModal;
  textarea: string;
}

export interface SetRejectModalIdIsShow {
  type: typeof SET_REJECT_MODAL_ID_ISSHOW;
  idAndIsShow: RejectModal;
}

export interface SetRejectModalTextarea {
  type: typeof SET_REJECT_MODAL_TEXTAREA;
  textarea: string;
}
