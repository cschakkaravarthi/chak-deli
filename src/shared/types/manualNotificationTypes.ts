import { ManualNotificationListModel, Categories, NotificationBell } from '../models/ManualNotification.model';
import Facet from '../models/Facet.model';

export const LOAD_MANUAL_NOTIFICATIONS = 'LOAD_MANUAL_NOTIFICATIONS';
export const LOAD_MANUAL_NOTIFICATIONS_FILTER = 'LOAD_MANUAL_NOTIFICATIONS_FILTER';
export const FETCH_MANUAL_NOTIFICATIONS = 'FETCH_MANUAL_NOTIFICATIONS';
export const FETCH_MANUAL_NOTIFICATIONS_FILTER = 'FETCH_MANUAL_NOTIFICATIONS_FILTER';
export const SET_MANUAL_NOTIFICATIONS_FILTER_FACET = 'SET_MANUAL_NOTIFICATIONS_FILTER_FACET';
export const LOAD_MANUAL_NOTIFICATIONS_BELL = 'LOAD_MANUAL_NOTIFICATIONS_BELL';
export const FETCH_MANUAL_NOTIFICATIONS_BELL = 'FETCH_MANUAL_NOTIFICATIONS_BELL';
export const PUT_MANUAL_NOTIFICATIONS_ARCHIVE = 'PUT_MANUAL_NOTIFICATIONS_ARCHIVE';
export const SET_MANUAL_NOTIFICATIONS_IS_NEW_OR_ARCHIVED = 'SET_MANUAL_NOTIFICATIONS_IS_NEW_OR_ARCHIVED';

export interface ManualNotificationListState {
  manualNotificationsListState: ManualNotificationListModel;
  facets: {};
  categories: Categories[];
  selectedFacet: string;
  notificationBell: NotificationBell;
  isNewOrArchived: string;
}
export interface LoadManualNotificationsAction {
  type: typeof LOAD_MANUAL_NOTIFICATIONS;
  notificationsList: ManualNotificationListModel;
}
export interface LoadManualNotificationsFilterAction {
  type: typeof LOAD_MANUAL_NOTIFICATIONS_FILTER;
  filterList: Categories[];
}

export interface SetFilterFacetAction {
  type: typeof SET_MANUAL_NOTIFICATIONS_FILTER_FACET;
  selectedFacet: Facet;
}
export interface LoadManulNotificationsBellAction {
  type: typeof LOAD_MANUAL_NOTIFICATIONS_BELL;
  NotificationBell: NotificationBell;
}

export interface SetIsNewOrArchived {
  type: typeof SET_MANUAL_NOTIFICATIONS_IS_NEW_OR_ARCHIVED;
  isNewOrArchived: string;
}
