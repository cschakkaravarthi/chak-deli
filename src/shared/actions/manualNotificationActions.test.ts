import moxios from 'moxios';
import { dummyManualNotificationListModel } from '../../shared/models/ManualNotification.model';
import { loadManualNotifications, fetchManualNotifications } from './manualNotificationAction';
import { LOAD_MANUAL_NOTIFICATIONS } from '../types/manualNotificationTypes';
import getStore from '../services/mockGlobalStore';

describe('notificationActions', () => {
  let store: any;
  beforeEach(() => {
    store = getStore();
    moxios.install();
  });

  afterEach(() => {
    moxios.uninstall();
  });

  it('creates a loadArticles action', () => {
    expect(loadManualNotifications(dummyManualNotificationListModel)).toMatchSnapshot();
  });

  it.skip('fetchNotifications makes API call', done => {
    // Skipping this test until response is working
    const expectedActions = [LOAD_MANUAL_NOTIFICATIONS];

    moxios.stubRequest('notifications', {
      status: 200,
      responseText: 'OK'
    });

    store
      .dispatch(fetchManualNotifications(''))
      .then(() => {
        const actionsCalled = store
          .getActions()
          .map((action: any) => action.type);
        expect(actionsCalled).toEqual(expectedActions);
        done();
      })
      .catch((error: any) => {
        done(error);
      });
  });
});
