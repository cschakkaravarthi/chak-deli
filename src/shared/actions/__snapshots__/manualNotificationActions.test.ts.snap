// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`notificationActions creates a loadArticles action 1`] = `
Object {
  "notificationsList": Object {
    "alerts": Object {
      "notifications": Array [
        Object {
          "body_full": "This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of!",
          "categories": Array [
            Object {
              "drupal_id": 1,
              "title": "Categories",
            },
          ],
          "created": 1571873331,
          "isNew": true,
          "notificationType": "maintenence",
          "summary": "This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of!",
          "title": "Emergency Alert - Los Angeles Area Earthquake",
          "type": "notification",
        },
        Object {
          "body_full": "This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of!",
          "categories": Array [
            Object {
              "drupal_id": 1,
              "title": "Categories",
            },
          ],
          "created": 1571873331,
          "isNew": true,
          "notificationType": "maintenence",
          "summary": "This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of!",
          "title": "Emergency Alert - Los Angeles Area Earthquake",
          "type": "notification",
        },
      ],
      "total": 0,
    },
    "approvals": Object {
      "notifications": Array [
        Object {
          "body_full": "This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of!",
          "categories": Array [
            Object {
              "drupal_id": 1,
              "title": "Categories",
            },
          ],
          "created": 1571873331,
          "isNew": true,
          "notificationType": "maintenence",
          "summary": "This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of!",
          "title": "Emergency Alert - Los Angeles Area Earthquake",
          "type": "notification",
        },
        Object {
          "body_full": "This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of!",
          "categories": Array [
            Object {
              "drupal_id": 1,
              "title": "Categories",
            },
          ],
          "created": 1571873331,
          "isNew": true,
          "notificationType": "maintenence",
          "summary": "This is a notification about a very important event that you should be aware of! This is a notification about a very important event that you should be aware of!",
          "title": "Emergency Alert - Los Angeles Area Earthquake",
          "type": "notification",
        },
      ],
      "total": 0,
    },
  },
  "type": "LOAD_MANUAL_NOTIFICATIONS",
}
`;
